lib/internal/errors.js

```diff
@@ -748,6 +748,8 @@ E('ERR_BUFFER_OUT_OF_BOUNDS',
 E('ERR_BUFFER_TOO_LARGE',
   `Cannot create a Buffer larger than 0x${kMaxLength.toString(16)} bytes`,
   RangeError);
+E('ERR_CALLS_UNAVAILABLE_DURING_EXIT', 'Cannot call Tracker.calls() in ' +
```

Comment:
I would call this just `ERR_UNAVAILABLE_DURING_EXIT` and drop anything in the error message specifically related to this API. That way, the error can potentially be reused for other things in the future.